### ISSUE TITLE
[cmake] Configure error with cmake 3.19.1

### ISSUE THREAD

**Issue by JackBoosY** [coin3d/simage#42]
Hi guys,
When I use cmake 3.19.1 configure simage, it shows:
```
-- Configuring done
CMake Error: install(EXPORT "simage" ...) includes target "simage" more than once in the export set.
-- Generating done
```

See https://github.com/coin3d/simage/blob/29d34eda95167f4e101d12e55394ddc731f03582/CMakeLists.txt#L663-L674

It export the target `${PROJECT_NAME}` twice.

Related: https://github.com/microsoft/vcpkg/pull/14716

**Comment by VolkerEnderlein** [comment]
Hi Jack,
thanks for notifying us. As this has worked for at least one and a half year without hassle I've started on the CMake discourse forum a [thread](https://discourse.cmake.org/t/error-when-installing-targets-and-exports-in-3-19-1/2245) as I think it's a regression in CMake. Let's see what the maintainers suggest and what further action we need to take.
Cheers, Volker

**Comment by VolkerEnderlein** [comment]
Provided proper fix with PR #43
